tests/apollo/CMakeLists.txt

```diff
@@ -31,60 +31,60 @@ endif()
 
 foreach(STORAGE_TYPE ${STORAGE_TYPES})
   add_test(NAME skvbc_basic_tests_${STORAGE_TYPE} COMMAND sh -c
-          "env ${APOLLO_TEST_ENV} STORAGE_TYPE=${STORAGE_TYPE} python3 -m unittest test_skvbc ${TEST_OUTPUT}"
+          "env ${APOLLO_TEST_ENV} STORAGE_TYPE=${STORAGE_TYPE} TEST_NAME=skvbc_basic_tests_${STORAGE_TYPE} python3 -m unittest test_skvbc ${TEST_OUTPUT}"
```

Comment:
This `TEST_NAME` variable seems redundant... Let's drop it. See my comment about how you can retrieve the test module's name.